src/Context.tsx

```diff
@@ -21,10 +23,20 @@ const { Provider, Consumer: RestfulReactConsumer } = React.createContext<Restful
   requestOptions: {},
 });
 
+let originalBase: string = "";
```

Comment:
Storing this so that the `originalBase` prop can not get overridden by using input. It is set internally by the constructor and passed in through contexted